src/tasks/keycloak/errors.ts

```diff
@@ -0,0 +1,17 @@
+import { HttpError as K8sHttpError } from '@kubernetes/client-node'
+import { HttpError as KeyCloakHttpError } from '@linode/keycloak-client-node'
+
+export class WrappedError extends Error {}
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export function extractError(operationName: string, error: any): any {
```

Comment:
Did using the Error object like this not work? Then we could also remove the eslint line above. And the less `any` the better :) 
```suggestion
export function extractError(operationName: string, error: Error): Error {
```

Reply:
Good suggestion, thanks!